src/app/search/search-filters/dupes.tsx

```diff
@@ -233,13 +234,29 @@ function computeStatDupeLower(
   for (const item of armor) {
     if (item.stats && item.power && item.bucket.hash !== BucketHashes.ClassArmor) {
       const statsToConsider = customStats[item.classType] ?? armorStats;
-      statsCache.set(
-        item,
-        _.sortBy(
-          item.stats.filter((s) => statsToConsider.includes(s.statHash)),
-          (s) => s.statHash
-        ).map((s) => s.base)
-      );
+
+      // Artifice armor can be +3 in any one stat, so we insert a separate version of the item for
+      // each stat considered
+      if (isArtifice(item)) {
+        for (const stat of statsToConsider) {
+          statsCache.set(
+            item,
+            item.stats
+              .filter((s) => statsToConsider.includes(s.statHash))
+              .sort((a, b) => a.statHash - b.statHash)
+              // One stat gets +3
+              .map((s) => (s.statHash === stat ? s.base + 3 : s.base))
+          );
+        }
```

Comment:
This ends up overwriting the entry in the map five times, no?

Reply:
Oh, I mixed up this intermediate cache with the stats-set itself